js/form.js

```diff
@@ -31,26 +31,95 @@ const onCheckOutChange = () => {
   checkIn.value = checkOut.value;
 }
 
-typeField.addEventListener('change', onTypeChange);
+const onTypeChange = () => {
+  priceField.placeholder = MIN_PRICES[typeField.value];
+  priceField.min = MIN_PRICES[typeField.value];
+}
 
 checkIn.addEventListener('change', onCheckInChange);
 
 checkOut.addEventListener('change', onCheckOutChange);
 
+typeField.addEventListener('change', onTypeChange);
+
+// Валидация заголовка
+const checkFormTitle = () => {
+  const valueLength = formTitle.value.length;
+
+  if (valueLength < MIN_NAME_LENGTH) {
+    formTitle.setCustomValidity(`Ещё ${MIN_NAME_LENGTH - valueLength} симв.`);
+  } else if (valueLength > MAX_NAME_LENGTH) {
+    formTitle.setCustomValidity(`Удалите лишние ${valueLength - MAX_NAME_LENGTH} симв.`);
+  } else {
+    formTitle.setCustomValidity('');
+  }
+
+  formTitle.reportValidity();
+}
+
+formTitle.addEventListener('input', checkFormTitle);
+
+// Валидация цены
+const onPriceInput = () => {
+  const type = typeField.value;
+  const price = priceField.value;
+  const minPrice = MIN_PRICES[type];
+
+  if (price < minPrice) {
+    priceField.setCustomValidity(`Стоимость должна быть не менее ${minPrice}`);
+  } else if (price > MAX_PRICE_PER_NIGHT) {
+    priceField.setCustomValidity(`Стоимость не должна превышать ${MAX_PRICE_PER_NIGHT}`);
+  } else {
+    priceField.setCustomValidity('');
+  }
+}
+
+priceField.addEventListener('input', onPriceInput);
+
+// Валидация комнат и гостей
+const checkNumberRooms = () => {
+  const roomsValue = parseInt(roomNumber.value, 10);
```

Comment:
можно еще так привести к типу Number
```suggestion
  const roomsValue = +roomNumber.value
```